.azure/pipelines/richnav.yml

```diff
@@ -0,0 +1,108 @@
+#
+# See https://docs.microsoft.com/en-us/vsts/pipelines/yaml-schema for details on this file.
+#
+
+# Configure which branches trigger builds
+trigger:
+  branches:
+    include:
+    - blazor-wasm
+    - master
+    - release/*
+    - internal/release/*
+
+variables:
+- name: DOTNET_SKIP_FIRST_TIME_EXPERIENCE
+  value: true
+- name: _TeamName
+  value:  AspNetCore
+- name: _BuildArgs
+  value: '/p:SkipTestBuild=true'
+- name: _PublishArgs
+  value: ''
+# Write binary logs for all main Windows build steps except the x86 one in public and PR builds.
+- name: WindowsArm64LogArgs
+  value: /bl:artifacts/log/Release/Build.arm64.binlog
+- name: Windows64LogArgs
+  value: /bl:artifacts/log/Release/Build.x64.binlog
+- name: Windows86LogArgs
+  value: -ExcludeCIBinaryLog
+- name: WindowsInstallersLogArgs
+  value: /bl:artifacts/log/Release/Build.Installers.binlog
+- name: WindowsArm64InstallersLogArgs
+  value: /bl:artifacts/log/Release/Build.Installers.Arm64.binlog
+- name: _UseHelixOpenQueues
+  value: 'true'
+- name: _SignType
+  value: ''
+- name: _InternalRuntimeDownloadArgs
+  value: ''
+- name: _InternalRuntimeDownloadCodeSignArgs
+  value: ''
```

Comment:
Remove all unused variables. Suspect only `$(Windows64LogArgs)` `$(Windows86LogArgs)` and `$(WindowsInstallersLogArgs)` are needed in the list after `$(_BuildArgs)`